doc/classes/AnimationPlayer.xml

```diff
@@ -217,7 +219,8 @@
 			<argument index="0" name="reset" type="bool" default="true">
 			</argument>
 			<description>
-				Stop the currently playing animation. If [code]reset[/code] is [code]true[/code], the anim position is reset to [code]0[/code].
+				Stop the currently playing animation. If [code]reset[/code] is [code]true[/code], the animation position is reset to [code]0[/code] and the playback speed is reset to 1.0.".
```

Comment:
There's a trailing `".` here (sorry :D).

Reply:
And might be good to use `[code]1.0[/code]` for consistency with `[code]0[/code]`.

Reply:
Done.